docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,107 @@
+# Move an instance from one Cleura Cloud region to another
+
+This guide will show you how to move or copy your instance to a different region in Cleura cloud.
+
+## Prerequisites
+
+- Installed OpenStack [CLI client](https://docs.cleura.cloud/howto/getting-started/enable-openstack-cli/)
```

Comment:
Use a relative link here:

```markdown
[OpenStack CLI](../getting-started/enable-openstack-cli/)
```

Reply:
Had to use another type of path then mentioned above but got it to work.